packages/app/vite.config.ts

```diff
@@ -1,3 +1,3 @@
 import config from '../../configurations/vite.config'
 
-export default config(__dirname)
+export default config(__dirname, false)
```

Comment:
Boolean flags are something of an antipattern. Maybe we can use an options object with a descriptive name for the option?

```ts
interface ConfigOptions {
  poolStrategy: 'threads' : 'forks'
}

export default function config(packagePath: string, {poolStrategy = 'threads'}: ConfigOptions = {}) {
  // ...
  pool: poolStrategy,

// in this file
export default config(__dirname, {poolStrategy: 'forks'})
```